redis-queue-transport.js

```diff
@@ -34,8 +35,13 @@ module.exports = function (options) {
     var type = args.type
     var listen_options = seneca.util.clean(_.extend({}, options[type], args))
 
-    var redis_in = Redis.createClient(listen_options.port, listen_options.host)
-    var redis_out = Redis.createClient(listen_options.port, listen_options.host)
+    if (_.isUndefined(listen_options.url)) {
+      var redis_in = Redis.createClient(listen_options.port, listen_options.host)
```

Comment:
please declare `redis_in` and `redis_out` outside of the conditional, then assign within the conditional.